collect_app/src/main/java/org/odk/collect/android/utilities/FileUtils.java

```diff
@@ -488,12 +488,10 @@ public static Bitmap getBitmap(String path, BitmapFactory.Options originalOption
     }
 
     public static byte[] read(File file) {
-        byte[] bytes = {};
-        try {
-            bytes = new byte[(int) file.length()];
-            InputStream is = new FileInputStream(file);
+        byte[] bytes = new byte[(int) file.length()];
+        try (InputStream is = new FileInputStream(file)) {
             is.read(bytes);
-            is.close();
+            return bytes;
```

Comment:
I think this return is redundant.

Reply:
We are using try-with-resources, so it auto closes the stream

Reply:
Got it! What about `return`?

Reply:
Yes you are right about the return statement. I'll remove it